### DISCUSSION TITLE
Questions about the function "round2Miller".

### DISCUSSION THREAD

**Discussion by WWJ1588** [mtex-toolbox/mtex#2104] [Ask Anything] [answered]
When I used the "round2Miller" function to convert Euler angle into Miller index, I got the result of "(8 -7 12)[-1107]". It is difficult for me to tell whether [-1107] means [-1 10 7] or [-11 0 7]. Is there any way to avoid this situation? 

`CS = crystalSymmetry('Cmcm', [6.163 9.728 4.704], 'mineral');`
`orientation = orientation.byEuler(60*degree, 30*degree, 120*degree, CS); `
`miller_index = round2Miller(orientation, CS);`

**Comment by kilir** [comment, marked as answer]
Hi,
with just one output argument, round2Miller returns a string.
You could have it comma separated
`miller_index = round2Miller(orientation, CS,'commasep')`

or you could have two output arguments and in return get Millers as a result.
`[d,p] = round2Miller(orientation, CS)`

Hope that helps,

Rüdiger

**Reply by WWJ1588** [reply]
Thank you very much！！！
